embassy-stm32/src/lib.rs

```diff
@@ -48,6 +48,8 @@ pub mod crc;
 ))]
 pub mod flash;
 pub mod pwm;
+#[cfg(all(quadspi, feature = "unstable-pac"))]
```

Comment:
this shouldn't be gated on `unstable-pac`

Reply:
Unfortunately I don't think it can be passed without `unstable-pac` feature, as the module uses `crate::gpio::low_level::AFType` which without `unstable-pac` cannot resolve `low_level` module and `AFType` from it is used in configuration here: https://github.com/Mirror0/embassy/blob/ad4970ae03190a6c966fed2c4d4a72b196cd9f1c/embassy-stm32/src/qspi/mod.rs#L103

Have you have any suggestion that I can use to avoid this dependencies? Because as far as I can see, only this prevents the build without `unstable-pac`

Reply:
you can use `crate::gpio::sealed` which is always available (within the crate)